packages/notifier/src/publish-kit.js

```diff
@@ -298,6 +304,13 @@ const advanceDurablePublishKit = (context, value, targetStatus = 'live') => {
   }
   const { tailP: currentP, tailR: resolveCurrent } =
     provideDurablePublishKitEphemeralData(state, facets);
+  const commit = resolution => {
+    // Invoke a direct callback, but ignore the result.
+    if (onUpdate) {
+      void callE(onUpdate, Promise.resolve(resolution));
```

Comment:
Thinking about this, we should comment on why `Promise.resolve()` is safe and required here: it's only used on values we created in this file, and we need to keep the rejection promise as is (through `Promise.resolve()`'s brand check) to avoid losing the quiet part of the rejection.